src/main/java/org/brapi/v2/api/VariantsetsApiController.java

```diff
@@ -638,20 +635,20 @@ ProgressIndicator getProgress() {
 		@Override
 		public void run() {
 			try {
-				String[] splitId = variantSet.getVariantSetDbId().split(GigwaGa4ghServiceImpl.ID_SEPARATOR);
+				String[] splitId = variantSet.getVariantSetDbId().split(Helper.ID_SEPARATOR);
 	        	int projId = Integer.parseInt(splitId[1]);
 				String module = splitId[0];
 				Document varQuery = new Document("$and", new BasicDBList() {{ add(new Document("_id." + VariantRunDataId.FIELDNAME_PROJECT_ID, projId)); add(new Document("_id." + VariantRunDataId.FIELDNAME_RUNNAME, splitId[2])); }} );
 				
 				MongoTemplate mongoTemplate = MongoTemplateManager.get(module);
-				result = exportHandler.createExportFiles(module, varColl.getNamespace().getCollectionName(), varQuery, variantSet.getVariantCount(), new ArrayList<>(), new ArrayList<>(), exportId, new HashMap<>(), new HashMap<>(), samplesToExport, progress);
+				result = exportHandler.createExportFiles(module, null /*FIXME*/, varColl.getNamespace().getCollectionName(), varQuery, variantSet.getVariantCount(), new ArrayList(), new ArrayList<>(), exportId, new HashMap(), new HashMap<>(), samplesToExport, progress);
```

Comment:
I'm a bit confused, because it doesn't look like the signature of `createExportFiles` changed that recently. But if the signature didn't recently changed, then I don't understand how the code was even building previously.